### PR TITLE
Add setup check for background blur

### LINKED ISSUE #6503: Add setup check for background blur + documentation
- [ ] There should be a setup check in `settings/admin/overview` or `settings/admin/talk`
- [ ] There should be a comment in our docs: https://nextcloud-talk.readthedocs.io/en/latest/system-requirements/#webserver
- [ ] The default setup in Nextcloud docs should be adjusted like in https://github.com/nextcloud/documentation/pull/930/files

### PR DESCRIPTION
**Pull request by danxuliu**
Fixes #6503 

**Reviewers:** this adds new strings! Also feel free to change them as needed, because I am not convinced by the texts I wrote :-P

Pending:
- [ ] Ensure that there are no memory leaks caused by the workers used to do the checks
- [ ] Is there a better (non hacky) way to load the workers from the administration settings page?
- [X] Add the documentation

### CODE REVIEW

File: src/components/AdminSettings/WebServerSetupChecks.vue [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+			// instead, so it fails to load.
+			//
+			// "publicPath" option in "worker-loader" configuration does not
+			// work with Webpack 5. As a workaround the public path needs to be
+			// overriden at runtime before loading the worker and restored
+			// afterwards.
+			// https://github.com/webpack-contrib/worker-loader/issues/281
+			const __webpack_public_path__saved = __webpack_public_path__
+
+			__webpack_public_path__ = __webpack_public_path__.replace('talk', 'spreed')
```
Comment (nickvergessen) [inline src/components/AdminSettings/WebServerSetupChecks.vue:153]: I got this in my console:

> GET https://nextcloud24.local/apps/spreed/js/talk-JitsiStreamBackgroundEffect.worker.worker.js?v=1472aec776b99b3ad271
[HTTP/1.1 404 Not Found 0ms]
> 10:54:01.876 Laden des Workers von "https://nextcloud24.local/apps/spreed/js/talk-JitsiStreamBackgroundEffect.worker.worker.js?v=1472aec776b99b3ad271" wurde auf Grund eines nicht freigegebenen MIME-Typs ("text/html") blockiert.

And a never finishing loading spinner.

For me the path should be `/appspriority/spreed/js/`

The following code (with importing the function from nextcloud/router package) works:
```suggestion
			__webpack_public_path__ = generateFilePath('spreed', 'js', '')
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
Time to fixup and "ready to review"?

**Comment by danxuliu** [comment]
> Time to fixup and "ready to review"?

I knew that I was forgetting something... :facepalm:

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
Lets backport to 23 not 13 :P

**Pull request merged**